libkbfs/md_id_journal.go

```diff
@@ -101,11 +100,25 @@ func (j mdServerBranchJournal) readMdID(r MetadataRevision) (MdID, error) {
 
 // All functions below are public functions.
 
-func (j mdServerBranchJournal) journalLength() (uint64, error) {
-	return j.j.journalLength()
+func (j mdIDJournal) length() (uint64, error) {
+	return j.j.length()
 }
 
-func (j mdServerBranchJournal) getHead() (MdID, error) {
+func (j mdIDJournal) getEarliest() (MdID, error) {
+	earliestRevision, err := j.readEarliestRevision()
+	if err != nil {
+		return MdID{}, err
+	} else if earliestRevision == MetadataRevisionUninitialized {
+		return MdID{}, nil
+	}
+	mdID, err := j.readMdID(earliestRevision)
```

Comment:
Any reason not to `return j.readMdID(earliestRevision)` as you do below in reading the latest version?